src/vehicle_move.cpp

```diff
@@ -671,7 +671,7 @@ veh_collision vehicle::part_collision( int part, const tripoint &p,
             int dam = obj_dmg * dmg_mod / 100;
 
             // No blood from hallucinations
-            if( !critter->is_hallucination() ) {
+            if( critter == nullptr && !critter->is_hallucination() ) {
```

Comment:
Unless I'm reading this wrong, this will check for "no critter" and if this "no critter" isn't a hallucination? Maybe you meant `critter != nullptr`?

Reply:
Yes, thanks!